controllers/loginController.js

```diff
@@ -1,73 +1,41 @@
-var bodyParser=require('body-parser');
-module.exports=function(app){
-/* MONGOOSE SETUP */
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://username:password@databaseserver:port/databasename',{ useNewUrlParser: true});
-/*Replace the above connection string with the actual connection string of your MongoDB database*/
-const Schema = mongoose.Schema;
-const UserDetail = new Schema({
-      username: String,
-      password: String
+
+exports.getSuccessLogin = function(req, res) {

```

Comment:
try and export all. your functions at once  on the last line, it's easier to read for everyone :) 
Ex: 
```javascript
module.exports = { functionA, functionB, functionEtc } 
```

Reply:
> try and export all. your functions at once on the last line, it's easier to read for everyone :)
> Ex:
> 
> ```js
> module.exports = { functionA, functionB, functionEtc } 
> ```

Stilul folosit de mine l-am luat dintr-un curs de pe Udemy, dar o sa fac modificarea sugerata de tine.